src/ucs/arch/aarch64/cpu.h

```diff
@@ -51,14 +56,12 @@ BEGIN_C_DECLS
 #define ucs_memory_bus_load_fence()   ucs_aarch64_dmb(oshld)
 
 /* The macro is used to flush all pending stores from write combining buffer.
- * Some uarch "auto" flush the stores once cache line is full (no need for additional barrier).
+ * Micro-arch "auto" flush the stores once WC buffer is full or on a timeout.
+ * DGH instruction hint provides a formal way to flush the gather buffer.  See
+ * rdma subsystem patch:
+ * https://patchwork.kernel.org/project/linux-rdma/patch/0-v1-c5dade92f363+11-mlx5_io_stop_wc_jgg@nvidia.com/
  */
-#if defined(HAVE_AARCH64_THUNDERX2)
```

Comment:
can we remove HAVE_AARCH64_THUNDERX2 from configure?

Reply:
It is in few other places, like TX2 memcpy, etc. So, we have to keep it around.